include/RAJA/exec-cuda/raja_cuda.hxx

```diff
@@ -65,6 +65,12 @@
 namespace RAJA
 {
 
+#if defined(RAJA_ENABLE_CLANG_CUDA)
```

Comment:
Yeah, my hope is we can push this into a bug in the Clang bug tracker and remove it, but for now we need it. It's also broken: that's one of the things we want to discuss